src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -104,10 +105,27 @@ doclet.see.class_or_package_not_accessible=Tag {0}: reference not accessible: {1
 doclet.see.nested_link=Tag {0}: nested link
 doclet.tag.invalid_usage=invalid usage of tag {0}
 doclet.Deprecated_API=Deprecated API
+doclet.Deprecated_Elements=Deprecated {0}
+doclet.Deprecated_In_Release=Deprecated in {0}
+doclet.Deprecated_Tabs_Intro=(The leftmost tab "Deprecated ..." indicates all the \
+    deprecated elements, regardless of the releases in which they were deprecated. \
+    Each of the other tabs "Deprecated in ..." indicates the elements deprecated \
+    in a specific release.)
```

Comment:
This is OK for now, but a noreg-doc follow-up might be to improve the punctuation in this text, possibly after consulting with docs folk.

It's the use of the quoted strings that seems weird, without any commas or parentheses too set them off. On the other hand, I realize we sometimes go for brevity in places like this.